### DISCUSSION TITLE
Max485 module/chip

### DISCUSSION THREAD

**Discussion by thrystan** [tagyoureit/nodejs-poolController#762] [RS485]
Has anyone ever gotten the max485 chip or one of the max485 modules to work with this? I think the issue is setting the correct gpio pin(s) to use for the xmt/rcv enable pins. I'm building my own raspberry pi "hat" and I'd like to have the rs485 interface on the board rather than using a separate usb interface.

**Comment by rstrouse** [comment]
If I am not mistaken that is the chip that is used on the building automation card.
https://cdn.shopify.com/s/files/1/0534/4392/0067/files/BuildingAutomation-Schematic_57338cbb-f0a7-474a-91f1-9e1a3c4bbb7a.pdf?v=1676868685

**Comment by thrystan** [comment]
Interesting, it looks like they are triggering the enable pins based on the transmit pin. I'll have to try that. I've seen various python scripts online that use one or two of the gpio pins to handle the enable pins on the chip.

**Reply by rstrouse** [reply]
I think that may have something to do with the RS485 pass-thru vs MODBUS mode.

**Comment by thrystan** [comment]
Ok, I'll try that approach. I hadn't been able to find as clear of a schematic for that as the one you linked.
